wasmbinding/message_plugin.go

```diff
@@ -985,7 +985,7 @@ func (m *CustomMessenger) addSchedule(ctx sdk.Context, contractAddr sdk.AccAddre
 		})
 	}
 
-	err := m.CronKeeper.AddSchedule(ctx, addSchedule.Name, addSchedule.Period, msgs)
+	err := m.CronKeeper.AddSchedule(ctx, addSchedule.Name, addSchedule.Period, msgs, crontypes.ExecutionStage(crontypes.ExecutionStage_value[addSchedule.ExecutionStage]))
```

Comment:
bindings handler shouldn't access Keepers directly, the interaction should be done via module's message server. otherwise we skip the validation step which is done on the message server's layer. that's not your fault but ab old bug. could you please replace cron keeper direct usage with its message server for all occasions in this file?

Reply:
Done